### PR TITLE
MH-13617 Re-introduce Prepare AV

### PR DESCRIPTION
**Pull request by KatrinIhler**
The Select-Tracks WO replaced prepare-av at parts of the standard workflow where it doesn't make sense. This also archives the media created by prepare-av so they don't have to be re-created.

~This PR includes #952.~

_This work was sponsored by SWITCH._

### CODE REVIEW

File: etc/org.opencastproject.adminui.cfg [hunk 1]
```diff
@@ -203,14 +203,14 @@
 # The default workflows use presenter/source, so that's the recommended default value.
 #
 # Default: presenter/source
```
Comment (gregorydlogan) [inline etc/org.opencastproject.adminui.cfg]: Default no longer matches setting

File: etc/org.opencastproject.adminui.cfg [hunk 2]
```diff
@@ -203,14 +203,14 @@
...
-#sourcetrack.left.flavor=presenter/source
+sourcetrack.left.flavor=presenter/prepared
 
 # Flavor that identifies the right hand stream, for use in the "Tracks" and "Thumbnail" panels of the video editor.
 #
 # The default workflows use presentation/source, so that's the recommended default value.
 #
 # Default: presentation/source
```
Comment (gregorydlogan) [inline etc/org.opencastproject.adminui.cfg]: Default no longer matches setting

File: etc/workflows/partial-publish.xml [hunk 3]
```diff
@@ -10,6 +10,17 @@
 
   <operations>
 
+    <operation
+        id="select-tracks"
+        exception-handler-workflow="partial-error"
+        description="Preparing presenter (camera) audio and video work versions">
+      <configurations>
+        <configuration key="source-flavor">*/prepared</configuration>
+        <configuration key="target-flavor">*/work</configuration>
```
Comment (gregorydlogan) [inline etc/workflows/partial-publish.xml:19]: So `/work` is used in this partial, and no where else?

**Review by gregorydlogan — CHANGES_REQUESTED** [review]

File: etc/workflows/partial-publish.xml (hunk 3 above)
Comment (KatrinIhler) [reply etc/workflows/partial-publish.xml:19]: True. I guess we could rename it, but I couldn't think of something better...

File: etc/org.opencastproject.adminui.cfg (hunk 2 above)
Comment (KatrinIhler) [reply etc/org.opencastproject.adminui.cfg]: Fixed.

File: etc/org.opencastproject.adminui.cfg (hunk 1 above)
Comment (KatrinIhler) [reply etc/org.opencastproject.adminui.cfg]: Thanks, fixed!

**Comment by KatrinIhler** [comment]
@gregorydlogan Any more comments for this one?

File: etc/workflows/partial-publish.xml (hunk 3 above)
Comment (gregorydlogan) [reply etc/workflows/partial-publish.xml:19]: Derp, I missed that this was used as source for other things in this partial.  I was expected it to be used in *other* partials...

**Review by gregorydlogan — APPROVED** [review]

**Comment by gregorydlogan** [comment]
I'll wait for this report to finish, but looking at the error I'm pretty sure it's some weird transient with the admin ui (sigh).  Once that's done I'll merge this in.

**Pull request merged**